src/Project.js

```diff
@@ -77,10 +78,10 @@ export default class Project {
         let match = packagesByName[depName];
         if (!match) continue;
 
-        let actual = depVersion.replace(/^\^/, '');
+        let actual = depVersion;
```

Comment:
Could I make a not, and ask that we rename any usages where we are talking about ranges, to have range in the name? 

Would that be a huge change? It wasn't done we had to think about before, but would be useful